romancal/regtest/test_mos_pipeline.py

```diff
@@ -22,7 +22,7 @@ def run_mos(rtdata_module, resource_tracker):
     rtdata.get_asn("WFI/image/L3_regtest_asn.json")
 
     # Test Pipeline
-    output = "r0099101001001001001_F158_visit_coadd.asdf"
+    output = "r0000101001001001001_f158_coadd.asdf"
```

Comment:
We don't expect to make this kind of product in operations; this is a non-skycell coadd.  So we don't have a defined naming scheme here.  I needed to change it to make F158 lowercase, and elected to drop the _visit_ since it's somewhat implied by the fact that the initial string only goes down to the visit level.